homeassistant/components/lutron_caseta/cover.py

```diff
@@ -78,3 +64,93 @@ async def async_set_cover_position(self, **kwargs: Any) -> None:
         if ATTR_POSITION in kwargs:
             position = kwargs[ATTR_POSITION]
             await self._smartbridge.set_value(self.device_id, position)
+
+
+class LutronCasetaTiltOnlyBlind(LutronCasetaDeviceUpdatableEntity, CoverEntity):
+    """Representation of a Lutron tilt only blind."""
+
+    _attr_supported_features = (
+        CoverEntityFeature.OPEN_TILT
+        | CoverEntityFeature.CLOSE_TILT
+        | CoverEntityFeature.STOP_TILT
+        | CoverEntityFeature.SET_TILT_POSITION
+        | CoverEntityFeature.OPEN_TILT
+    )
+    _attr_device_class = CoverDeviceClass.BLIND
+
+    @property
+    def is_closed(self) -> bool:
+        """Return if the cover is closed."""
+        return self._device["tilt"] == 0 or self._device["tilt"] == 100
+
+    @property
+    def current_cover_tilt_position(self) -> int:
+        """Return the current position of cover."""
+        return self._device["tilt"]
+
+    async def async_stop_cover_tilt(self, **kwargs: Any) -> None:
+        """Top the cover."""
+        await self._smartbridge.stop_cover(self.device_id)
```

Comment:
There is some duplicate code here. Please refactor the `LutronCasetaCover` to have a base class so it can share so we don't have duplicates

Reply:
actually i think this wasn't correct anyway, stop and tilt stop are separate methods